test/system/700-play.bats

```diff
@@ -196,6 +196,30 @@ EOF
     run_podman rm -a
 }
 
+@test "podman kube play read-only" {
+    YAML=$PODMAN_TMPDIR/test.yml
+    run_podman create --pod new:pod1 --name test1 $IMAGE touch /test
+    run_podman create --pod pod1 --read-only --name test2 $IMAGE touch /test
+    run_podman create --pod pod1 --read-only --name test3 $IMAGE touch /tmp/test
+    run_podman kube generate pod1 -f $YAML
```

Comment:
Can we check the output to verify that read-only field is properly set?

Reply:
Fixed.